apps/bare-expo/scripts/start-metro.sh

```diff
@@ -21,6 +21,15 @@ yarn start --port ${port}
 EOF
   # execute the file in a new command line window
   chmod 0755 ${commandFile}
-  open ${commandFile}
-  
+  case "$OSTYPE" in
+     darwin*)
+        open ${commandFile}
+        ;;
+     *)
+       # Spawns a new terminal window and detaches it
+       # Assuming the non-standard variable $TERMINAL is set
+        nohup "$TERMINAL" -e "${commandFile}" </dev/null >/dev/null 2>&1 &
```

Comment:
> should we throw an error if `$TERMINAL` is not set? we may also want to use `EXPO_` prefix here, eg: `$EXPO_TERMINAL` in order to avoid clashing with other tools

(just explaining my decision) I thought about using that prefix but I chose to use $TERMINAL as it seems that some other applications use it too and I thought it was a good idea since the user wouldn't have to create an additional one for expo. I've seen many dotfiles on github exporting TERMINAL is what I'm saying.